src/api/app/views/webui2/webui/staging/workflows/_create_staging_project.html.haml

```diff
@@ -6,8 +6,8 @@
           %h5.modal-title#create-staging-projects-modal-label Create Staging Project
         .modal-body
           .form-group
-            = label_tag(:staging_project_name, "#{staging_workflow.project}:Staging:")
-            = text_field_tag(:staging_project_name, nil, placeholder: 'Eg: A', class: 'form-control')
+            = label_tag(:staging_project_name, 'Please enter a project you have permissions to')
```

Comment:
Not sure if this label makes sense anymore. I need a second opinion here:smile:

Reply:
It doesn't really, what about making this an autocomplete for `Project.for_user|Project.for_group`?

Reply:
This project doesn't exist yet. This form would create it.

Reply:
Yes but you can only create projects below `Project.for_user|Project.for_group` :-)

Reply:
I also find this confusing. What about: `Enter project name (you need permissions to create it)` or just `Enter project name` and if he doesn't have permissions he will get an error. :thinking: